plugins/destination/mssql/client/client_test.go

```diff
@@ -25,6 +34,9 @@ func TestPlugin(t *testing.T) {
 			return destination.NewPlugin("mssql", plugin.Version, New, destination.WithManagedWriter())
 		},
 		Spec{ConnectionString: getTestConnection()},
-		destination.PluginTestSuiteTests{},
+		destination.PluginTestSuiteTests{
+			MigrateStrategyOverwrite: migrateStrategy,
```

Comment:
I think we want to base not null on the not null field in column spec here (in this destination plugin) as well